### PR TITLE
Added basic documentation and policy example

### PR DESCRIPTION
**Pull request by olehermanse**

### CODE REVIEW

File: tom.cf [hunk 1]
```diff
@@ -0,0 +1,136 @@
+# Policy to update and run the Tom GitHub bot
+# Note: This is example policy, for production changes, go to private infra repo
+
+body file control
+{
+    inputs => { "$(sys.libdir)/stdlib.cf", "$(sys.libdir)/vcs.cf" };
```
Comment (nickanderson) [inline tom.cf]: I changed this to $(sys.local_libdir) to fix up the duplicate definition of bundle issues I had

File: README.md [hunk 2]
```diff
@@ -1,3 +1,74 @@
-# CFEngine Bot, Tom
+# Northern.tech Bot, Tom
 
 I'm your friendly neighbourhood bot, Tom.
+I trigger pull requests when someone mentions me in a GitHub Pull Request.
+
+## Usage
+
+## Command line
```
Comment (craigcomstock) [inline README.md]: Should Usage and Command line be different levels of hash-tags?

File: README.md [hunk 3]
```diff
@@ -1,3 +1,74 @@
...
+**Explanation:**
+
+* `GITHUB_TOKEN` - API token generated by GitHub, for the bot username, specified in [`config.json`](/config.json).
+* `JENKINS_USER` - Jenkins username (LDAP username in CFEngine Jenkins).
+* `JENKINS_TOKEN` - API token, generated in Jenkins Settings UI.
+* `JENKINS_CRUMB` - [GET `https://<jenkins>/crumbIssuer/api/xml`](https://stackoverflow.com/questions/16738441/how-to-request-for-the-crumb-issuer-for-jenkins) (for CFEngine: https://ci.cfengine.com/crumbIssuer/api/xml).
+
+## Technical details
+
+## Webhooks / polling
```
Comment (craigcomstock) [inline README.md]: same as Usage/Command line above, different levels of hash tags?

File: tom.cf [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+{
+   meta:
+     "tags"
+       slist => {"autorun"},
+       if => filesexist( @(config_files) );
+
+    vars:
+        "config_files" slist => {"/home/tom/config.json",
+                                 "/home/tom/TOM_ENABLE"};
+
```
Comment (craigcomstock) [inline tom.cf]: Any docs about TOM_ENABLE file maybe? Just delete that and tom is disabled right?

File: tom.cf [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+            "Update"
+                usebundle => "tom_update";
+            "Install"
+                usebundle => "tom_install";
+            "Run"
+                usebundle => "tom_run";
+    reports:
+        inform_mode.!has_tom_config::
+            "Skipping execution, config missing in /home/tom";
+}
```
Comment (craigcomstock) [inline tom.cf]: and this message could include notice that either config and/or TOM_ENABLE is missing.

File: README.md (hunk 2 above)
Comment (olehermanse) [reply README.md]: Yup, don't know why that happened.

File: tom.cf (hunk 4 above)
Comment (olehermanse) [reply tom.cf]: Sure.

File: README.md (hunk 3 above)
Comment (olehermanse) [reply README.md]: Thanks.

Comment (olehermanse) [reply README.md]: They are not hash tags :) Number sign, hash, or pound sign.

**Pull request merged**